assessment.sql

```diff
@@ -5,24 +5,44 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (
+   id SERIAL PRIMARY KEY,
+   surname CHARACTER VARYING,
+   given_name CHARACTER VARYING,
+   gender CHARACTER VARYING,
+   height INTEGER,
+   weight INTEGER,
+   dob DATE,
+   eye_color CHARACTER VARYING,
+   hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+  VALUES ( 'Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black' )
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT COUNT(*), eye_color, hair_color
```

Comment:
>Write a query to get all attributes of licensees...

This means get all columns. How do we get all the columns of each row we select?